src/ethereum/rfc003/mod.rs

```diff
@@ -3,3 +3,15 @@ pub mod ether_htlc;
 
 pub use erc20_htlc::Erc20Htlc;
 pub use ether_htlc::EtherHtlc;
+
+// ascii of "redeemed"
+pub const REDEEMED_LOG_MSG: &str =
+    "72656465656d6564000000000000000000000000000000000000000000000000";
+// ascii of "refunded"
+pub const REFUNDED_LOG_MSG: &str =
+    "726566756e646564000000000000000000000000000000000000000000000000";
+// ascii of "tooEarly"
+pub const TOO_EARLY: &str = "0x746f6f4561726c79000000000000000000000000000000000000000000000000";
```

Comment:
Why is this one prefixed with `0x` and the other ones are not?

Reply:
No particular reason :) Updated it so that they are all the same.